notebooks/featurestore/hsfs/serving/feature_engineering-online-fs.ipynb

```diff
@@ -0,0 +1,88 @@
+{
+ "cells": [
+  {
+   "cell_type": "raw",
+   "metadata": {},
+   "source": [
+    "---\n",
+    "title: \"Ingestion to Online Feature Store\"\n",
+    "date: 2021-02-24\n",
+    "type: technical_note\n",
+    "draft: false\n",
+    "---"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import hsfs\n",
+    "connection = hsfs.connection()\n",
+    "fs = connection.get_feature_store()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from hops import hdfs\n",
+    "from pyspark.sql import functions as F\n",
+    "\n",
+    "stores_csv = spark.read\\\n",
+    "             .option(\"inferSchema\", \"true\")\\\n",
+    "             .option(\"header\", \"true\")\\\n",
+    "             .format(\"csv\")\\\n",
+    "             .load(\"hdfs:///Projects/{}/Jupyter/hsfs/archive/stores data-set.csv\".format(hdfs.project_name()))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "online_store_fg_meta = fs.create_feature_group(name=\"online_store_fg\",\n",
+    "                                       version=1,\n",
+    "                                       primary_key=['store'],\n",
+    "                                       description=\"Store related features\",\n",
+    "                                       online_enabled=True,\n",
+    "                                       time_travel_format=None,\n",
+    "                                       statistics_config={\"enabled\": True, \"histograms\": True, \"correlations\": True})"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "online_store_fg_meta.save(stores_csv)"
+   ]
+  }
+ ],
+ "metadata": {
+  "kernelspec": {
+   "display_name": "Python 3",
+   "language": "python",
+   "name": "python3"
+  },
+  "language_info": {
+   "codemirror_mode": {
+    "name": "ipython",
+    "version": 3
+   },
+   "file_extension": ".py",
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.6.9"
+  }
+ },
+ "nbformat": 4,
+ "nbformat_minor": 4
+}
```

Comment:
This notebook make no sense by itself. It's just a bunch of code with no explanation. Plus we already have a notebook for online serving - it's here: https://github.com/logicalclocks/hops-examples/blob/master/notebooks/featurestore/hsfs/feature_vector_model_serving.ipynb - Please get rid of this one.

Reply:
The examples site should not just contain long-form notebooks. It should be a quick page to find answers to problems you have. How do i write a csv training dataset in 1 file? 
They are HOW-TO docs. We have a mix of tutorials and HOW-TO. You cannot say they should be just tutorials - that would defeat the purpose of them.

We are missing a notebook for "How do i ingest into the online FS". That is a different question from "How do i do online feature serving". We need to answer more questions. If the code is self explanatory, we should not need reems of (tutorial) text around it. We shouldn't have to show online feature vector serving if all I want to see is how to ingest into the online FS.